bob-apps/bin/bobp.rs

```diff
@@ -736,20 +819,20 @@ async fn get_worker(net_conf: NetConfig, task_conf: TaskConfig, stat: Arc<Statis
         let res = if measure_time {
             let start = Instant::now();
             let res = client.get(request).await;
-            stat.save_single_thread_get_time(&start.elapsed());
+            stat.get.common.save_single_thread_time(&start.elapsed());
```

Comment:
`get_worker` only needs `GetStatistics` so it can receive `Arc<GetStatistics>` as parameter (not the whole statistics). The same for other simple worker functions